arcade/gui/property.py

```diff
@@ -1,6 +1,19 @@
 import sys
 import traceback
-from typing import Any, Callable, Generic, Optional, Set, TypeVar, cast
+
+from typing import (
+    Optional,
+    TypeVar,
+    Generic,
+    Tuple,
+    Callable,
+    Any,
+    Iterable,
+    Dict,
+    List,
+    Set,
```

Comment:
```suggestion
    Set,
    SupportsIndex,
```
You're missing this import & it's causing an undefined symbol error lower down in the file.